corehq/apps/reports/v2/utils.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+import dateutil
+import pytz
+from memoized import memoized
+
+from corehq.const import SERVER_DATETIME_FORMAT
+from corehq.util.dates import iso_string_to_datetime
+from corehq.util.timezones.conversions import PhoneTime
+from corehq.util.timezones.utils import get_timezone_for_user
+
+
+@memoized
+def get_timezone(request, domain):
```

Comment:
This seems like it belongs in `timezones.utils`

Reply:
fixed